src/rill.h

```diff
@@ -117,27 +122,23 @@ const char * rill_store_file(const struct rill_store *store);
 unsigned rill_store_version(const struct rill_store *store);
 rill_ts_t rill_store_ts(const struct rill_store *store);
 size_t rill_store_quant(const struct rill_store *store);
-size_t rill_store_keys(const struct rill_store *store);
-size_t rill_store_vals(const struct rill_store *store);
+size_t rill_store_keys_count(const struct rill_store *store, enum rill_col column);
 size_t rill_store_pairs(const struct rill_store *store);
+size_t rill_store_index_len(const struct rill_store *store, enum rill_col col);
+
 
+struct rill_pairs *rill_store_query_value(
```

Comment:
Again we may want to go for `rill_store_query(..., enum rill_col col);`

Reply:
The idea was to hide the col in the public interface, so just provide two functions that do it for you. I can remove them, and expose the main function though. Let me know!